tutorials/3d/baked_lightmaps.rst

```diff
@@ -80,28 +96,28 @@ It can be found in the Mesh menu:
 This will generate a second set of UV2 coordinates which can be used for baking,
 and it will also set the texture size automatically.
 
-Unwrap on scene import
-~~~~~~~~~~~~~~~~~~~~~~
+Unwrap from your 3D DCC
+~~~~~~~~~~~~~~~~~~~~~~~
 
-This is probably the best approach overall. The only downside is that, on large
-models, unwrapping can take a while on import.
-Just select the imported scene in the filesystem dock, then go to the Import tab.
-There, the following option can be modified:
+The last option is to do it from your favorite 3D app. This approach is generally
+not recommended, but it's explained first so that you know it exists.
```

Comment:
The "but it's explained first" bit should be removed now unwrapping on import is explained first.